Formula/lunarvim.rb

```diff
@@ -0,0 +1,36 @@
+class Lunarvim < Formula
+  desc "IDE layer for Neovim. Completely free and community driven"
+  homepage "https://www.lunarvim.org"
+  url "https://github.com/LunarVim/LunarVim-mono/archive/refs/tags/stable.tar.gz"
```

Comment:
This seems to just download the repo at https://github.com/LunarVim/LunarVim. We should use the original repo instead.

Reply:
We use this mono repo to simplify the process to install package with `cmake` and handle some issues related to the path.

Reply:
Sure, but this is not appropriate for a formula in Homebrew/core. For one thing, we require URLs to be versioned, but yours just points to a floating `stable` tag. This means that the tarball checksum can change at any time.

Reply:
So it would be the best if we moved the cmake setup to the main LunarVim/LunarVim repo then? If so I'll do it, it will probably help with packaging for other package managers as well

Reply:
Yes, since that is the source of the code